src/terrain/services/teams.clj

```diff
@@ -36,9 +37,15 @@
 (defn join [{user :shortUsername} name]
   (ipg/join-team user name))
 
-(defn join-request [{user :shortUsername user-name :commonName email :email :as user-info} name message]
+(defn join-request [{user :shortUsername user-name :commonName email :email} name message]
```

Comment:
The `:as user-info` bit was actually debugging code because I was logging the user information. This variable is no longer used, so I removed it.